.github/workflows/release-firmware-release-config.yml

```diff
@@ -0,0 +1,46 @@
+name: "[Release] firmware release config"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-message-release:
```

Comment:
this should be renamed

Reply:
Thanks! done https://github.com/trezor/trezor-suite/pull/18435/files#diff-45639c8f4273030d4c3ce8b0a6245bb2e38ed51f8ca515f30bb12594d43e61e7R15